src/runtime/virtcontainers/pkg/types/pcipath.go

```diff
@@ -0,0 +1,98 @@
+// Copyright Red Hat.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+package types
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+const (
+	pciSlotBits = 5
+	maxPciSlot  = (1 << pciSlotBits) - 1
```

Comment:
Might be worth a comment stating these need to match the equivalents in `src/agent/src/pci.rs`.

Reply:
Well, it's not really that they need to match each other, but they both need to be correct w.r.t. the PCI spec.  I'll put in a comment making it clearer these values aren't arbitrary.